docs/bottom-tab-navigator.md

```diff
@@ -6,18 +6,31 @@ sidebar_label: createBottomTabNavigator
 
 A simple tab bar on the bottom of the screen that lets you switch between different routes. Routes are lazily initialized -- their screen components are not mounted until they are first focused.
 
-To use this navigator, you need to install [`react-navigation-tabs`](https://github.com/react-navigation/react-navigation-tabs) and its peer dependencies:
+To use this navigator, you need to install `@react-navigation/bottom-tabs` and its peer dependencies:
 
 ```sh
 npm install react-navigation-tabs react-native-reanimated react-native-gesture-handler
 ```
 
-To use this tab navigator, import it from `react-navigation-tabs`:
+To use this tab navigator, import it from `@react-navigation/bottom-tabs`:
 
 ```js
-import { createBottomTabNavigator } from 'react-navigation-tabs';
-
-createBottomTabNavigator(RouteConfigs, TabNavigatorConfig);
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+
+const Tab = createBottomTabNavigator();
+function App() {
+  return (
+    <NavigationContainer>
+      <Tab.Navigator initialRouteName="feed">
+        <Tab.Screen name="feed" component={Feed} />
+        <Tab.Screen name="article" component={Article} />
+        <Tab.Screen name="notifications">
+          {props => <Notifications {...props} />}
```

Comment:
We should prolly avoid this render callback in general docs. Only keep it for advanced cases